### PR TITLE
[Feature] Origami Layout

### PR DESCRIPTION
**Pull request by Sqyphen**
## Description
Adds Origami layout to the Barcode URL builder page

## Implementation
Uses Origami components to build a a page similar to Polyfill.io and Image service. Mistakenly done to bring the page in line with Internal services but it does make the page more readable.

## Screenshot
screenshot 2019-01-28 at 17 08 06

## .env changes
None

## Additional requirements
None

### CODE REVIEW

File: views/index.hbs [hunk 1]
```diff
@@ -1,97 +1,261 @@
...
+              </fieldset>
+            </div>
+          </div>
+        </div>
+
+        
+
+        
+
+        <!--
```
Comment (Lily2point0) [inline views/index.hbs]: Please don't leave big commented out chunks of code

Comment (Sqyphen) [reply views/index.hbs]: Fixed!

File: views/index.hbs [hunk 2]
```diff
@@ -1,97 +1,219 @@
...
+          Example image URL:
+        </p>
+        <p>
+          <a target="_blank" href="/barcode?&dateFrom=2018-11-13&dateTo=2018-11-14&orientation=v&fit=fill&order=colour&sort=asc">/barcode?dateFrom=2018-11-13&amp;dateTo=2018-11-14&amp;orientation=v&amp;fit=fill&amp;order=colour&amp;sort=asc</a>
+        </p>
+      </div>
+
+      <div class="o-layout__main" data-o-component="o-syntax-highlight">
+
+        <h3>URL Generator</h3>
```
Comment (Lily2point0) [inline views/index.hbs]: Should be `h2` to be semantic html

Comment (Sqyphen) [reply views/index.hbs]: Done :)

**Pull request merged**